src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM.java

```diff
@@ -254,7 +251,9 @@ public String getSshCredentialsId() {
     }
 
     public List<GitSCMExtension> getExtensions() {
-        return gitSCM.getExtensions();
+        return gitSCM.getExtensions().stream()
+                .filter(e -> !(e instanceof InternalBitbucketRepositoryExtension))
```

Comment:
We need to filter _out_ the extension so that it doesn't show in the UI, otherwise it would show in the 'extensions' section when you're editing your job config

Reply:
Ah sorry I misread